docs/04-api-reference/front-commerce-remix/index.mdx

```diff
@@ -344,3 +344,8 @@ This should be added as a script in the `package.json`
   "worker": "front-commerce worker"
 }
 ```
+
+## Environment variables
+
+- `FRONT_COMMERCE_COOKIE_MAX_AGE_IN_SECONDS` - Default: 1 day. The cookie max
+  age in seconds.
```

Comment:
**issue:** This page already contains lots of things. I'm not in favor of adding another section

**suggestion:** create a new `Environment variables` page, similar to what was done for some extensions 
![2024-08-02_12-29](https://github.com/user-attachments/assets/080aa839-0542-476f-856b-2f1929e63fc6)

Reply:
I moved it to its own page